lib/trento_web/openapi/v1/schema/cluster.ex

```diff
@@ -37,13 +36,11 @@ defmodule TrentoWeb.OpenApi.V1.Schema.Cluster do
         site: %Schema{type: :string},
         hana_status: %Schema{type: :string},
         attributes: %Schema{
-          title: "ClusterNodeAttributes",
-          type: :array,
-          items: %Schema{type: :string}
+          type: :object,
```

Comment:
what kind of object can this be? I guess a `key => value` of type `string => string`.
Could er document it via the `additionalProperties` or something like that?